ArjunaJTA/jakartaee_transactional_app/README.md

```diff
@@ -9,11 +9,10 @@ bean and then test it using testsuite deployed in WildFly using Arquillian
 
 This command will build and test the application using Arquillian
 
-Get a WildFly that has the expected version of Narayana in it. If you need to build it:
+Get a WildFly that has the expected version of Narayana in it. If you need to obtain it:
```

Comment:
+1 to them being independent, but if what you proposed renders the instructions well I can change to that as I do hope that even if independent that they might typically be expected to be cloned or viewed via the quickstart repo rather than as separate folders.

Reply:
Incorporated in some manner this feedback, thanks